docs/reference/javascript-cheatsheet.md

```diff
@@ -0,0 +1,82 @@
+# JavaScript Cheatsheet
+
+In n8n, you can write custom JavaScript code snippets to add, remove, and update the data you receive from a node. You can also use code snippets to modify the data structure of the data returned by a node.
+
+**Note:** If you're using a code snippet in an expression in a node, wrap them in parenthesis. For example, to set the current timestamp in the Set node, use the following code snippet in the expression editor.
+
+```js
+{{Date.now();}}
+```
+
+## Date and Time
+
+The JavaScript [Date](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) object is a built-in object that stores the date and time. It provides several methods for managing and formatting the date. By default, the date and time are returned in the local (i.e. host system) time zone. 
```

Comment:
Let's say my system has CEST as the timezone but my n8n instance or my workflow has GMT set as the timezone. Which one will override which?